tests/test_productfetcher.py

```diff
@@ -10,7 +10,7 @@
 sys.modules['eups'] = Mock()
 sys.modules['eups.tags'] = Mock()
 
-from lsst.ci.models import DEFAULT_BRANCH_NAME
+from lsst.ci.models import DEFAULT_BRANCH_NAME # NOQA
```

Comment:
Normally we say `# noqa: E123` or whatever the error code is -- it self documents the reason. But I see that none of the other code does that and this is only a test file so 🤷